src/handlers/schedule-checkin.ts

```diff
@@ -205,7 +206,7 @@ function isRequestBody(value: any): value is RequestBody {
 
 interface ResponseBody {
   data: {
-    checkin_available_epoch: number;
-    checkin_boot_epoch: number;
+    checkin_available_epoch: number[];
+    checkin_boot_epoch: number[];
```

Comment:
I think an array of objects would be easier to consume by the frontend than parallel arrays of numbers.

Reply:
Agreed!